src/copy.rs

```diff
@@ -142,6 +142,12 @@ pub struct Options {
     /// that certain apps may have issues pasting when this option is used, in particular XWayland
     /// clients are known to suffer from this.
     serve_requests: ServeRequests,
+
+    /// Offer additional text mime types if at least one text mime type is provided.
+    ///
+    /// Additionaly offers `text/plain;charset=utf-8`, `text/plain`, `STRING`, `UTF8_STRING` and
+    /// `TEXT` mime types if at least one text mime type is provided.
+    offer_additional_text_mimes: bool
```

Comment:
This will default to `false`, however I'd like to preserve the current behavior. I guess you should convert the Default derive into a manual impl and set this field to true. Alternatively, reword this to invert the meaning -- something like `omit_additional_text_mime_types`.

Reply:
What about wrapping this specific field in a struct that defaults to true?

Reply:
Why? If we want to keep this as true = previous, then implementing Default manually will be the easiest.